apps/web/env.mjs

```diff
@@ -47,8 +47,10 @@ export const env = createEnv({
     INVITE_DISABLED: z.enum(["1", "0"]).optional(),
     IS_FORMBRICKS_CLOUD: z.enum(["1", "0"]).optional(),
     VERCEL_URL: z.string().optional(),
-    SURVEY_BASE_URL: z.string().url().optional(),
-    SHORT_SURVEY_BASE_URL: z.string().url().optional().or(z.string().length(0)),
+    SURVEY_URL: z.object({
+      SURVEY_BASE_URL: z.string().url().optional(),
+      SHORT_SURVEY_BASE_URL: z.string().url().optional().or(z.string().length(0)),
+    }).refine((obj) => obj.SHORT_SURVEY_BASE_URL !== obj.SURVEY_BASE_URL),
```

Comment:
Please remove the SURVEY_URL env as it creates confusion and we'd currently like to go with an easier approach